src/docs/implementations/dynatrace.adoc

```diff
@@ -175,6 +175,11 @@ management.metrics.export.dynatrace:
             key1: "value1"
             key2: "value2"
 
+        # (from 1.9.0) Wheter or not to use the Dynatrace specific summary instruments. (default: true)
+        # This should only be used if problems with existing instrumentation were discovered after upgrading to 1.9.0
+        # Set to false, this will restore the previous (1.8.+) behavior for Timers and DistributionSummaries.
```

Comment:
```suggestion
        # Set to false, this will restore the previous (1.8.x) behavior for Timers and DistributionSummaries.
```
some might mix up 1.8.+ with 1.8+